### PR TITLE
Project 9 – Music Releases – Emma Lindell & Ida Näslund

### PR DESCRIPTION
**Pull request by emmajosefina**
Link to my deployed project: https://music-releases-ida-emma.netlify.app/

My reflections on how this week's project turned out: We faced many problems this week because of not understanding npm enough to use it. 

Things I'd like to have clarified or explained in more detail: So this is why we're a little bit late with this week's project, since we experienced problems with npm.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,12 +1,11 @@
...
-Replace this readme with your own information about your project.
+# Music Releases 🎶
 
-Start by briefly describing the assignment in a sentence or two. Keep it short and to the point.
+This week's project was all about getting started with React and build a "New Releases" music site. We created components by using data from Spotify and created a page which shows new album and single releases.
 
 ## The problem
 
-Describe how you approached to problem, and what tools and techniques you used to solve it. How did you plan? What technologies did you use? If you had more time, what would be next?
+The problems that we experienced this week had to do with npm install, node-modules and to deploy to Netlify. We got really stuck but eventuelly we found what the problem was.
```
Comment (JenFi72) [inline README.md:7]: Great that you solved it :)

File: code/src/App.js [hunk 2]
```diff
@@ -1,12 +1,39 @@
...
+              />
+
+             ))}
+   
+
+    </section>
     </div>
+     <Footer /> 
+    </main>
+
```
Comment (JenFi72) [inline code/src/App.js:37]: Clean and easy to follow - appropriate and descriptive names.

File: code/src/components/Album.js [hunk 3]
```diff
@@ -0,0 +1,55 @@
...
+           />
+
+        <span className="symbol-color">
+        { 
+          props.artists.length > 2 && index >= 0 && index < props.artists.length -2
+          ? ', '
+          : index === props.artists.length -2
+          ? ' & '
+          : ''
+        }
```
Comment (JenFi72) [inline code/src/components/Album.js:44]: Very elegant way to solve "issue" with several artist connected to one album. Keeps the code short and clean.

File: code/src/components/Artist.js [hunk 4]
```diff
@@ -0,0 +1,18 @@
...
+ // Creating a new component
+const Artist = (props) => {
+   return (
+
+      // <div className="artist">
+        <a className="linked-artist" href={props.artistLink} target="_blank" rel="noopener noreferrer"> 
+        {props.artistName}
+        </a>
+         // </div>
+
```
Comment (JenFi72) [inline code/src/components/Artist.js:13]: Neat as everything else. Feels like you've aimed at keeping it as short as possible - good job.

File: code/src/index.css [hunk 5]
```diff
@@ -1,13 +1,176 @@
...
+  padding: 0 20px;
+  filter: brightness(0) invert(1);
+  z-index: 1;
+}
+
+.symbol-color {
+  color: #a0a0a0;
+}
+
+/*---------- HOVER TEXT ------------*/
```
Comment (JenFi72) [inline code/src/index.css:98]: Nice with comment, makes it easier to follow :)

File: code/src/index.css [hunk 6]
```diff
@@ -1,13 +1,176 @@
...
+  transition: 0.5s ease;
+  color: #fff;
+  text-decoration: underline;
+}
+
+.album-name:hover {
+  transition: 0.5s ease;
+  color: #fff;
+  text-decoration: underline;
+}
```
Comment (JenFi72) [inline code/src/index.css:109]: Very small comment, but maybe you could have merged these into one to "save up" code (as they have the same specifics). For instance:

.linked-artist, .album-name: hover{
transition: 0.5s ease;
  color: #fff;
  text-decoration: underline;
}

File: code/src/index.css [hunk 7]
```diff
@@ -1,13 +1,176 @@
...
+
+  .page {
+    display: flex;
+    flex-direction: row;
+    flex-wrap: wrap;
+    justify-content: space-around;
+  }
+}
+
+/*---------- MEDIA QUERY – DESKTOP ---------------*/
```
Comment (JenFi72) [inline code/src/index.css:162]: Very structured and easy to follow - looks great when viewing it.

**Review by JenFi72 — COMMENTED** [review]
Very great job Emma and Ida! I think you really put great effort in keeping the code neat and clean without excessive divs. CSS is very well structured and cohesive with title describing next step. The deployed site looks good in different device moods and you definitely reach blue level requirements. I understood you hade problems deploying it at Netlify - good job finding issues and be able to solve it! :)

File: code/src/App.js (hunk 2 above)
Comment (Kyparn) [reply code/src/App.js:37]: Everything is clean and easy to read.

File: code/src/components/Footer.js [hunk 8]
```diff
@@ -0,0 +1,13 @@
...
+const Footer = () => {
+    return (
+        <footer className="footer">
+            <span>Project music releases made by:</span>
+            <span> Emma Lindell & Ida Näslund</span>
+            <span>On Technigo Bootcamp spring 2022</span>
+        </footer>
+    )
+}
+
```
Comment (Kyparn) [inline code/src/components/Footer.js:12]: Nice that u guys added a footer, keeps the code cleaner

File: code/src/index.css [hunk 9]
```diff
@@ -1,13 +1,176 @@
+@import url("https://fonts.googleapis.com/css2?family=Raleway:wght@100;200;300;400;500;600;700&display=swap");
+
+* {
+  padding: 0;
+  margin: 0;
+}
```
Comment (Kyparn) [inline code/src/index.css:6]: You can always put this in your body tag insteed, now u have dubbel margin 0 :)

**Review by Kyparn — COMMENTED** [review]
Your code looks really nice, and also that u workt as a team makes it all better, super good job and keep it and, we are almost at the half way marker :)

**Pull request closed without merging**